docs-src/references/sdk-metrics.md

```diff
@@ -26,8 +26,11 @@ All metrics are prefixed with `temporal_` before being exported to their configu
 (The prefix has been removed in the following reference.)
 Currently, some metrics are specific to certain SDKs.
 
+TypeScript, Python, .NET, and Ruby SDKs metrics are defined in the Core SDK.
```

Comment:
```suggestion
TypeScript, Python, .NET, and Ruby SDKs metrics are defined in the Rust Core SDK.
```

Reply:
We don't. This is more of a general grammar question...and a bit philosophical. Using "defined by" overlaps with "defined as," such as "defined by its compact size," whereas "defined in" implies that the direct object is where you can find the definition.

The Rust Core SDK defines a lot more than just the SDK metrics, so in this instance using "defined by" seems to be overly broad. I recommend stetting (keeping "defined in").

Having said all that, the lines are fuzzy enough that I'm not gonna advocate strongly either way.